bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -99,8 +102,8 @@ val copyResources by tasks.register<Copy>("copyResources") {
         include("**/libmongocrypt.so", "**/libmongocrypt.dylib", "**/mongocrypt.dll")
     } else if (file(cmakeBuildPath).exists()){
         val jnaMapping = mapOf(
-                "libmongocrypt.so" to "linux-x86-64",
-                "mongocrypt.dll" to "win32-x86-64",
+                "libmongocrypt.so" to "linux-" + com.sun.jna.Platform.ARCH,
```

Comment:
Seemed the simplest way of using the same rules as JNA does to name the directory

Reply:
SGTM